hooks/pre-push

```diff
@@ -0,0 +1,8 @@
+#/usr/bin/env sh
+
+./test.sh
+
```

Comment:
Lines 4–8 (whilst a nice explanation of unix command exit statuses) are not necessary as the exit code of the last command run is the exit code of the script.